src/components/header/Header.component.tsx

```diff
@@ -1,2 +1,56 @@
+import cartWhite from '@/assets/img/headerImg/cartWhite.png';
+import login from '@/assets/img/headerImg/login.png';
+import logoMA from '@/assets/img/headerImg/logoMA.png';
+import moon from '@/assets/img/headerImg/moon.png';
+import signUp from '@/assets/img/headerImg/signUp.png';
+import sun from '@/assets/img/headerImg/sun.png';
+
 import styles from './header.module.css';
-export const HeaderComponent = () => <header className={styles.header}>Header</header>;
+export const HeaderComponent = () => (
+    <header className={styles.header}>
```

Comment:
<img width="1291" alt="Screenshot 2024-04-28 at 21 06 14" src="https://github.com/NosatskyiK/fe-react-2024/assets/115080771/9d361e6d-1035-4df1-ab99-64aca0cadd08">

Header and content are starting from the one line on design